### PR TITLE
Add allowed_capabilities_sys_admin Closes #516

### LINKED ISSUE #516: Add allowed_capabilities_sys_admin query for Kubernetes
### Platform
Kubernetes

### Description
Add a new "Allowed Capabilities Sys Admin" query, that checks if  CAP_SYS_ADMIN Linux capability is used

### PR DESCRIPTION
**Pull request by rafaela-soares**
For this query, it is necessary to check if SYS_ADMIN is a capability added.

Closes #516

### CODE REVIEW

File: assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "allowed_capabilities_sys_admin",
+  "queryName": "Allowed Capabilities Sys Admin",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "Containers should not have CAP_SYS_ADMIN Linux capability",
+  "descriptionUrl": "https://jamesdefabia.github.io/docs/user-guide/containers/"
```
Comment (ruigomescx) [inline assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json:7]: We should point to kubernetes official documentation if appliable. If not, leave empty

**Review by ruigomescx — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json (hunk 1 above)
Comment (rafaela-soares) [reply assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json:7]: I did not find a reference to "sys_admin" in the kubernetes official documentation. But I will point to the security context of the kubernetes official documentation

Comment (joaomartinscx) [reply assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json]: You could even be more specific in the section on that page, where [capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container) are mentioned. Right on the first line the is a link to manpages about [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html), which CAP_SYS_ADMIN is part of.

File: go.mod [hunk 2]
```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/uuid v1.1.2
 	github.com/hashicorp/hcl/v2 v2.7.0
 	github.com/mailru/easyjson v0.7.6
-	github.com/moby/buildkit v0.7.2
+	github.com/moby/buildkit v0.7.2 // indirect
```
Comment (joaomartinscx) [inline go.mod:13]: This doesn't seem necessary

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**